lib/containers/utils.pm

```diff
@@ -179,4 +179,9 @@ sub test_built_img {
     assert_script_run("rm -rf /root/templates");
 }
 
+sub is_sle_container_supported {
+    return 0 if check_os_release('opensuse', 'PRETTY_NAME') && is_sle;
+    return 1;
+}
```

Comment:
Can you explain what this actually does?

Reply:
the idea is to has some checks for no-supported host. For instance sle base on opensuse. We decided to run opensuse in OSD as we do for Centos and Ubuntu. For all those, the job runs with DISTRI and VERSION for sle as the image under test is sle. So you can read the condition as " if image under test is sle but host is opensuse" then the image is not supported.

Reply:
resolved as we agreed to implement another solution in the future.